src/index.html

```diff
@@ -4,10 +4,60 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+    <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars">
+      <div class="stars__block stars--0">
```

Comment:
modifier(stars--0) class should be used on corresponding block class

you do not need additional wrapper